server/opts.go

```diff
@@ -20,6 +20,7 @@ import (
 	"errors"
 	"flag"
 	"fmt"
+	certstore "github.com/nats-io/nats-server/v2/server/certstore"
```

Comment:
Don't need override and these usually go at the end with blank line separating them from stdlib items.